astropy/io/fits/hdu/table.py

```diff
@@ -1305,7 +1313,7 @@ def _load_data(cls, fileobj, coldefs=None):
         close_file = False
 
         if isinstance(fileobj, str):
-            fileobj = open(fileobj, 'r')
```

Comment:
not that I expect the default to change, but there is no harm in keeping the `'r'` around to be explicit.

Reply:
That's pyupgrade, but it makes since in py3 mode is a named parameter, which it was not before.